### PR TITLE
[chip, testplan] Update SYSRST_CTL and ADC

### PR DESCRIPTION
**Pull request by sriyerg**
- This commit addresses the comments and updates from the testplan
review meeting held on 6/07/2021.
- The meeting notes are below:
https://docs.google.com/document/d/1OhPP-HjciwKpIh0wWt1xqPqPf0Y0powmmww6xekwMeE/
- Updated SYSRST ctrl and ADC ctrl sections of the testplan

Signed-off-by: Srikrishna Iyer <sriyer@google.com>

### CODE REVIEW

**Review by msfschaffner — APPROVED** [review]
LGTM, but probably good if @esshiu can give this a pass as well.

**Review by udinator — APPROVED** [review]
LGTM, but will defer to @esshiu

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (matutem) [inline hw/top_earlgrey/data/chip_testplan.hjson:921]: Consistently use ctrl
```suggestion
            - Write the pin allowed ctrl register to allow some of the pins to be overridden with
```

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (matutem) [inline hw/top_earlgrey/data/chip_testplan.hjson:934]:
```suggestion
      desc: '''Verify the SYSRST ctrl can detect an input combination to signal an interrupt.
```

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (matutem) [inline hw/top_earlgrey/data/chip_testplan.hjson:952]: Why depend on coming from POR reset? This test could be run after some other test...

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (matutem) [inline hw/top_earlgrey/data/chip_testplan.hjson:957]: Why call it GSC? is it not a sysrst_ctrl wake up? Is it historical?

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (matutem) [inline hw/top_earlgrey/data/chip_testplan.hjson:974]: Same as above... Imagine a version of this test run within a collection of other tests.

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (matutem) [inline hw/top_earlgrey/data/chip_testplan.hjson:978]: Why should SW be idle for a reset?

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (matutem) [inline hw/top_earlgrey/data/chip_testplan.hjson:1059]:
```suggestion
              filter range. That should cause the pwrmgr to wake up.
```

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (sriyerg) [reply hw/top_earlgrey/data/chip_testplan.hjson:921]: Aah, I mimicked the design which uses `ctl` in the CSR name.

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (sriyerg) [reply hw/top_earlgrey/data/chip_testplan.hjson]: Fixed.

**Comment by esshiu** [comment]
Look good to me.

-Eric

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (esshiu) [reply hw/top_earlgrey/data/chip_testplan.hjson:1063]: It's indeed historical. I got the name (GSC) from Randall.

**Pull request merged**
